src/content/docs/Generic Programming/generics.md

```diff
@@ -13,27 +13,27 @@ For generic modules, the generic parameters follows the module name:
 
 ```c3
 // TypeA, TypeB, TypeC are generic parameters.
-module vector(<TypeA, TypeB, TypeC>);
+module vector{ TypeA, TypeB, TypeC };
```

Comment:
I found `module vector {TypeA, TypeB, TypeC}` is more natural. Use that instead.